code/modules/admin/admin_verbs.dm

```diff
@@ -905,6 +905,33 @@ var/list/fun_images = list()
 
 	M << browse(rules, "window=rules;size=800x1000")
 
+//"gonna add a specific button that shows a player a boilerplate "Your gimmick sucks ass and if you don't think so so do you" popup" - Warc 2022
+/client/proc/berate_player_gimmick(mob/M as mob in world) //(But it's actually Bat who did this)
+	set name = "Berate Player Gimmick"
+	set popup_menu = 0
+	SET_ADMIN_CAT(ADMIN_CAT_NONE)
+
+	var/crossness = input("What's wrong with this person's gimmick?", "Enter Crossness", "It's overplayed") as anything in list("It's overplayed", "It's just fucking horseshit", "Cancel")
+	if (!crossness || crossness == "Cancel")
+		return
+
+	if(!M.client)
+		alert("[M] is logged out, so you should probably ban them!")
+		return
```

Comment:
So this bit is a holdover from the show rules button which I adapted, IDK if this is appropriate here though.

Reply:
beautiful. thank you.